### PR TITLE
Cj4 0.2.1 pre release bug fixes

### PR DESCRIPTION
**Pull request by K20017**

### CODE REVIEW

**Comment by K20017** [comment]
1) Indented takeoff ref pages
2) Fixed landing weight code so ldgWtCell is updated with the actual landing weight instead of the fuel burned
3) Landing performance calculations are now done with the new computed landing weight instead of takeoff weight
4) Takeoff and landing weights turn yellow if they exceed 17110 (Max takeoff weight) and 15660 (max landing weight)

File: src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ class CJ4_FMC extends FMCMainDisplay {
         this._activatingDirectTo = false;
         this._templateRenderer = undefined;
         this._msg = "";
+		this.DSPLMENUcounter = 0;
```
Comment (dga711) [inline src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js:41]: Insteaf of keeping track of these values in fmc, wouldn't we want to always work off of the Simvar? Could make it a getter property that always reads from simvar

File: src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_DsplMenuPage.js [hunk 2]
```diff
@@ -1,21 +1,103 @@
...
+                fmc.DSPLMENUloNavaids = 1;
+            } else if (fmc.DSPLMENUloNavaids == 1) {
+				fmc.DSPLMENUcounter = fmc.DSPLMENUcounter - 64;
+				SimVar.SetSimVarValue("L:CJ4_MAP_SYMBOLS", "Number", fmc.DSPLMENUcounter);
+				fmc.DSPLMENUloNavaids = 0;
+            }
+            loNavaidsActive = fmc.DSPLMENUloNavaids == 0 ? "LO NAVAIDS[white][s-text]"
+            : "LO NAVAIDS[green]";
+				
+            fmc.clearUserInput();
```
Comment (dga711) [inline src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_DsplMenuPage.js:46]: no need to clear scratchpad input here
same for the others

File: src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_TakeoffRefPage.js [hunk 3]
```diff
@@ -66,15 +66,15 @@ class CJ4_FMC_TakeoffRefPage {
 
         fmc._templateRenderer.setTemplateRaw([
             [originIdent, "1/3[blue] ", "TAKEOFF REF[blue]"],
-            ["RWY ID[blue]", "WIND[blue]"],
+            [" RWY ID[blue]", "WIND[blue] "],
             [depRunwayOutput + "[s-text]", fmc.takeoffWindDir + "\xB0/" + fmc.takeoffWindSpeed],
```
Comment (dga711) [inline src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_TakeoffRefPage.js:70]: @cwburnett do we need to pad the values here?

File: src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_PerfInitPage.js [hunk 4]
```diff
@@ -370,7 +370,11 @@ class CJ4_FMC_PerfInitPage {
...
         }
-
+		
+		if (ldgWtCell > 15660) { //Turn the landing weight yellow if it exceeds the maximum landing weight
+			ldgWtCell = ldgWtCell + "[yellow]";
+		}
+		
         fmc._templateRenderer.setTemplateRaw([
             [destinationIdent, "2/3 [blue]", "APPROACH REF[blue]"],
             [" A/I[blue]"],
```
Comment (dga711) [inline src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_PerfInitPage.js:380]: @cwburnett do we need to pad the values here?

File: src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js [hunk 5]
```diff
@@ -38,6 +38,12 @@ class CJ4_FMC extends FMCMainDisplay {
...
+		this.DSPLMENUloNavaids = 0;
```
Comment (dga711) [inline src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js:42]: Insteaf of keeping track of these values in fmc, wouldn't we want to always work off of the Simvar? Could make it a getter property that always reads from simvar

**Review by dga711 — APPROVED** [review]

**Pull request merged**